indico/modules/events/papers/__init__.py

```diff
@@ -90,3 +91,11 @@ class PaperLayoutReviewerRole(ManagementRole):
     name = 'paper_layout_reviewer'
     friendly_name = _('Layout reviewer')
     description = _('Grants layout reviewing rights for assigned papers.')
+
+
+@signals.event.sidemenu.connect
+def _extend_event_menu(sender, **kwargs):
+
+    yield MenuEntryData(title=_("Call for Papers"), name='call_for_papers',
+                        endpoint='papers.call_for_papers', position=2,
```

Comment:
aren't we already using `position=2` for another menu entry? we could either add it to the end (need to check what the highest position is) or add it as `position=3` and increase all other positions >=3 by one. for now I'd go for `position=-1` (that adds it at the end) and check with @pferreir later if it's better to keep it at the end or insert it after CfA